File: 1200/1355B/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int n;
        cin >> n;
        vector<int> a(n);
        for (int i = 0; i < n; i++)
            cin >> a[i];

        sort(a.begin(), a.end());

        long long cnt = 0, grps = 0;
        for (int i = 0; i < n; i++) 
        {
            cnt++;
            if (cnt >= a[i]) 
            {
                grps++;
                cnt = 0;
            }
        }

        cout << grps << "\n";
    }
    
    return 0;
}

File: 1200/1355B/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            int n = sc.nextInt();
            int a[] = new int[n];
            
            for(int i=0;i<n;i++)
                a[i] = sc.nextInt();
            
            Arrays.sort(a);
            long cnt = 0;
            long grps = 0;
            
            for(int i=0;i<n;i++)
            {
                cnt++;
                if(cnt >= a[i])
                {
                    grps++;
                    cnt = 0;
                }
            }
            
            System.out.println(grps);
        }
    }
}